WordPress/gutenberg
# Block Editor: Rename UnstableRichTextInputEvent to convention

Related: #14239
Helps unblock: #15197

This pull request seeks to rename the `UnstableRichTextInputEvent` from `@wordpress/block-editor` to align to the convention of "Unstable & Experimental APIs" (i.e. renames to `__unstableRichTextInputEvent`).

>An experimental or unstable function or object should be prefixed respectively using `__experimental` or `__unstable`.

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis

**Testing Instructions:**

Verify there are no lingering references to the previous name.

Verify there is no impact on application runtime.
